editor/import/resource_importer_scene.cpp

```diff
@@ -1016,6 +1112,33 @@ bool ResourceImporterScene::get_internal_option_visibility(InternalImportCategor
 	return true;
 }
 
+bool ResourceImporterScene::get_internal_option_update_view(InternalImportCategory p_category, const String &p_option, const Map<StringName, Variant> &p_options) const {
```

Comment:
may use a different function like `get_internal_option_update_view_required` to make it a bit clearer.

Reply:
Yes, make sense, I'll change that

Reply:
Done!